showcase/image_analysis.md

```diff
@@ -0,0 +1,57 @@
+MBAT is a Python-based software tool that automates the ARG image analysis process, enabling researchers to focus on specific sub-regions of the brain. It includes a  Napari-based user interface that allows ARG slides of mouse brain tissue to be preprocessed and registered to topographical data from Allen Brain Atlas regions. This allows researchers to preprocess their ARG sections using different layout schemes contained in whole-slide images and automatically convert these sections into single images. The included user interface allows the user to match brain sections with the atlas and calculate mean background intensity. MBAT converts data from the Allen Institute’s Mouse Brain Atlas and stores the topographical information of various brain regions in a local database as regions of interest (ROIs), which can then be overlaid onto the ARG image. The user can translate, rotate, scale, and even edit brain region ROIs. The ARG signal for each ROI can be quantified and stored in a data file, enabling the user to process sections in smaller batches and resume the workflow later, rendering the workflow particularly user-friendly. The pipeline is availale for anyone to reuse on {% tool "github" %}.
+
+
+## What can you use the tool for?
+ 
+The Mouse Brain Alignment Tool provides an automated pipeline for analyzing brain autoradiography images. MBAT allows end-users to focus on image analysis without technical interruption, thereby increasing the reliability and reproducibility of results, thus expanding the possibilities of neurobiology research. The software is open source and packaged so that anyone, even without administrator access, can download and readily use the tool.
+The current iteration of MBAT is only applicable to ARG images of mouse brain, as it relies on the Mouse Brain Atlas. However, by incorporating additional standardized brain atlases, such as those for the rat animal model, it could be made more widely applicable to other brain analyses. Currently, it is only compatible with ARG scans; however, with minor modifications, ARG images set up on whole slides could be accommodated. With a few minor modifications, the tool can also be applied to a range of different image file types. In essence, the tool is useful for any application that aims to measure signals in coronal slices of the brain, whether through autoradiography or alternative staining methods, including fluorescent staining. Thus, the tool can greatly facilitate neuroinflammation research, whether induced through infectious diseases or other causes.
+
+## Resources
```

Comment:
I have a feeling with should impose a style guide here, now it looks a bit like "Miscellaneous section" :)